dist/pathfora.js

```diff
@@ -2581,11 +2581,11 @@ function setCustomColors (widget, colors) {
       contentUnit.style.setProperty('background-color', colors.actionBackground, 'important');
     }
 
-    if (colors.actionText) {
+    if (colors.actionText && contentUnitMeta.querySelector('h4')) {
```

Comment:
This is a tiny efficiency nit, but you can actually define this as a variable up in the jumble of vars at the top of this function, that saves us from doing the querySelector lookup twice.

```
var contentUnitMetaTitle = contentUnitMeta.querySelector('h4');

...

if (colors.actionText && contentUnitMetaTitle) {
  contentUnitMetaTitle.style.setProperty('color', colors.actionText, 'important');
}
```